examples/blog-basic/meta.yml

```diff
@@ -0,0 +1,16 @@
+# This file is used only to generate a browsable
+# list of examples for our documentation. You do
+# not need it in your own projects.
+meta:
+  tags:
+    - public        # works without Insiders
+      #- insiders      # requires Insiders edition
```

Comment:
Why the commented out tags? Tags were not used in any blog post. I understand that this is to configure the RSS plugin, but especially "insiders" + "requires the Insiders edition" might be confusing to readers.

Reply:
Ah, sorry, caused a misunderstanding here. The meta.yml contains information I thought would be useful to build index pages for the examples. LMK if you no like and I remove.

Reply:
Currently, the `transform` function does the following:

- Set the repo URL to the folder of the example
- Set the edit URI accordingly
- Set the copyright form the top-level `mkdocs.yml`
- Enable some features from the top-level `mkdocs.yml`
- Add icon configuration from the top-level `mkdocs.yml`

Reply:
Okay, now I understand the use case, pardon my stupidity. I think we should add them to the root of the repository, and ensure that they are not bundled with the downloadable example zips. We could then consume them in our transform function and/or a hook. Maybe we can start by calling it `.example.yml`, then it's clear that it's not MkDocs-related.